spaces/S000142/properties/P000065.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000142
+property: P000065
+value: true
+---
+
+Since $X\subseteq\mathbb Q^\omega$, we have $|X|\le\aleph_0^{\aleph_0}$.  On the other hand, $X\supseteq \prod_n([0,1/n]\cap\mathbb Q)$; hence $|X|\ge \aleph_0^{\aleph_0}$.  Therefore $|X|=\aleph_0^{\aleph_0}=\mathfrak c$.
```

Comment:
```suggestion
Since $X\subseteq\mathbb Q^\omega$, we have $|X|\le\aleph_0^{\aleph_0}$.  On the other hand, since $\sum_n\frac{1}{n^2}$ converges, $X\supseteq \prod_n([0,1/n]\cap\mathbb Q)$; hence $|X|\ge \aleph_0^{\aleph_0}$.  Therefore $|X|=\aleph_0^{\aleph_0}=\mathfrak c$.
```

(PS: this is a new trick I learned from the leanprover-community/mathlib4 community to suggest changes.) Use this button:

![image](https://github.com/pi-base/data/assets/1559632/81dbd42e-b128-47c0-ae50-9696cffac06d)

Reply:
Added your suggestion (with `1/n^2` instead of `\frac{1}{n^2}`, which resulted in something too small)